homeassistant/components/plant/__init__.py

```diff
@@ -111,7 +111,7 @@
 
 # Flag for enabling/disabling the loading of the history from the database.
 # This feature is turned off right now as its tests are not 100% stable.
-ENABLE_LOAD_HISTORY = False
+ENABLE_LOAD_HISTORY = True
```

Comment:
Is this being turned on only for tests or for production?
If the latter, we should remove the comment and the flag.

Reply:
fa2a7efd5f